components/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -0,0 +1,189 @@
+import React, { useState, useRef, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+
+import { theme as defaultTheme } from '../shared';
+import Icon from '../Icon';
+import Link from '../Link';
+
+const BreadcrumbsItems = styled.ol`
+  display: flex;
+  align-items: center;
+
+  list-style: none;
+
+  overflow: auto;
+
+  ${({
+    theme: {
+      spacing: { xxsmall, small },
+    },
+  }) => css`
+    padding: ${small}px ${xxsmall}px;
+  `}
+`;
+
+const BreadcrumbItem = styled.li`
+  display: flex;
+  align-items: center;
+
+  ${({
+    theme: {
+      baseFontSize,
+      colors: { neutral },
+      spacing: { xsmall },
+    },
+  }) => css`
+    font-size: ${baseFontSize * 0.75}px;
+
+    a {
+      display: flex;
+      align-items: center;
+
+      &:hover {
+        text-decoration: underline;
+      }
+    }
+
+    span {
+      color: ${neutral[700]};
+    }
+
+    & + li {
+      margin-left: ${xsmall}px;
+    }
+  `}
+`;
+
+const BreadcrumbIcon = styled(Icon).attrs(
+  ({ name = 'keyboard_arrow_right' }) => ({
+    name,
+    size: 'small',
+    'aria-hidden': true,
+  }),
+)`
+  ${({
+    theme: {
+      colors: { primary },
+      spacing: { xxsmall, xsmall },
+    },
+    name,
+  }) => css`
+    margin-right: ${name === 'home' ? xxsmall : xsmall}px;
+    color: ${primary[700]};
+  `}
+`;
+
+const ExpandButton = styled.button.attrs(() => ({
+  type: 'button',
+}))`
+  border: none;
+  background-color: transparent;
+
+  cursor: pointer;
+
+  &:hover {
+    font-weight: 700;
+  }
+
+  ${({
+    theme: {
+      colors: { primary },
+    },
+  }) => css`
+    color: ${primary[700]};
+  `}
+`;
+
+const Breadcrumbs = ({ items, theme }) => {
+  if (items.length < 2) {
+    throw new Error('Breadcrumbs items should be greater or equal than 2.');
+  }
```

Comment:
Instead of trowing a error, maybe hidding the element in case some problem happens? What do you think?

Reply:
I like to throw an error because the correction can be imediate. What do you think?

Reply:
At the end. I accepted your suggestion, because I couldn't to test this. I tried treat this with ErrorBoundary but it wasn't work and I didn't understand why. I'll research it in another time.